### PR TITLE
Fixed font, border around highlights, and functional rendering of button

### PR DESCRIPTION
**Pull request by briansai**
https://trello.com/c/8lJdvbLI
https://trello.com/c/RUeOO4jY
https://trello.com/c/HOeqyIhY
https://trello.com/c/zXo0eorG
https://trello.com/c/HHfv647H

### CODE REVIEW

**Comment by briansai** [comment]
@lighthurst  Can you please review this primitive start of the styling please?  Thanks!

File: client/src/components/descriptions/DescriptionEntry.jsx [hunk 1]
```diff
@@ -15,25 +15,23 @@ class DescriptionEntry extends React.Component {
     const desc = this.props.description;
 
     if (this.state.id === 0) {
-      this.setState({ description: desc.descriptions });
-    } else if (this.state.id === 1) {
       this.setState({ type: 'The Space' });
       this.setState({ description: desc.space });
```
Comment (lighthurst) [inline client/src/components/descriptions/DescriptionEntry.jsx]: Why not setState to both properties in one invocation?

File: client/src/components/guests/RoomEntry.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    };
+  }
+
+  componentDidMount() {
+    const room = this.props.room;
+
+    if (this.state.id === 0) {
+      this.setState({ image: '' });
+      this.setState({ room: 'guests' });
+      this.setState({ roomInfo: room.guests });
```
Comment (lighthurst) [inline client/src/components/guests/RoomEntry.jsx]: Add all three properties to one setState invocation

File: client/src/components/highlights/HighlightEntry.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    };
+  }
+
+  componentDidMount() {
+    const highlights = this.props.highlight;
+
+    if (this.state.id === 0) {
+      this.setState({ desc: 'Sparkling Clean' });
+      this.setState({ highlight: highlights.sparklingClean });
+      this.setState({ text: ' recent guests have said that this home was sparkling clean.' });
```
Comment (lighthurst) [inline client/src/components/highlights/HighlightEntry.jsx:23]: Add all three properties to one setState invocation.

File: client/src/components/highlights/HighlightsList.jsx [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
 const HighlightsList = props => (
   <div>
-    {props.highlightsList.map((highlights, index) => {
-        <Highlights key={index} highlights={highlights} />
-        console.log(highlights.sparklingClean);
-      })
-    }
+    <Container>
+      {props.highlightsList.map((highlight, index) => (
+          <HighlightEntry id={index} highlight={highlight} />
```
Comment (lighthurst) [inline client/src/components/highlights/HighlightsList.jsx]: I do the same thing, but AirBnB style guide doesn't like us adding an index for a key. If you think of anything else to use, let me know so I can incorporate in my code.

File: client/src/components/highlights/buttonList/HelpfulButton.jsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+const ButtonDetails = styled.svg`
+  height: 1em;
+  width: 1em;
+  display: block;
+  fill: currentcolor;
+  `;
+
+const HelpfulButton = (props) => {
+  const d = 'm8.37 1c-.34 0-.53.43-.64.79l-.04.17c-.17.74-.56 2.47-3.76 3.58-1.97.68-2.93 2.31-2.93 4.96 0 1.68.56 4.5 4.3 4.5h6.2c.3 0 1-.06 1-.63 0-.41-.26-.63-.5-.63a.5.5 0 1 1 0-1c .92 0 1-.31 1-.63 0-.52-.38-.61-.54-.63a.5.5 0 0 1 -.46-.52.5.5 0 0 1 .5-.48c1 0 1-.41 1-.63s0-.62-1-.62a.5.5 0 1 1 0-1c1 0 1-.47 1-.63 0-.58-.77-.63-1-.63h-4.5a1 1 0 0 1 -.83-1.56c.05-.07.16-.19.31-.35.34-.35.92-.93 1.21-1.58.24-.52.35-1.25.28-1.78 0-.01-.12-.74-.62-.74m3.15 15.04h-6.2c-3.32 0-5.3-2.06-5.3-5.5 0-3.09 1.21-5.08 3.61-5.91 2.67-.93 2.97-2.23 3.11-2.86.02-.08.04-.16.05-.22.37-1.31 1.16-1.51 1.6-1.51 1.02 0 1.52.96 1.61 1.61.09.71-.06 1.64-.37 2.32-.37.81-1.02 1.47-1.41 1.86-.09.1-.17.17-.2.21h4.5c1.2 0 2 .65 2 1.63 0 .46-.16.84-.45 1.11.39.37.45.86.45 1.14 0 .34-.09.98-.74 1.35.15.23.24.53.24.9 0 .41-.12 1.01-.74 1.36.15.26.24.56.24.89 0 .97-.8 1.63-2 1.63';
```
Comment (lighthurst) [inline client/src/components/highlights/buttonList/HelpfulButton.jsx]: Nice!

**Review by lighthurst — APPROVED** [review]
Nice use of styled components. Just a few changes, but approved. Keep up the good work!

File: client/src/components/descriptions/DescriptionEntry.jsx (hunk 1 above)
Comment (briansai) [reply client/src/components/descriptions/DescriptionEntry.jsx]: Thanks Aaron!! Good point!

File: client/src/components/guests/RoomEntry.jsx (hunk 2 above)
Comment (briansai) [reply client/src/components/guests/RoomEntry.jsx]: Ditto above! and all the other ones that say this!

**Pull request merged**